### PR TITLE
[Geeks for Geeks Solution]: Fascinating Number

### PR DESCRIPTION
**Pull request by iamanolive**
resolves issue #345

### CODE REVIEW

File: dsa-solutions/gfg-solutions/0002-fascinating-number.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+Given a number N. Your task is to check whether it is fascinating or not.
+
+Fascinating Number: When a number(should contain 3 digits or more) is multiplied by 2 and 3, and when both these products are concatenated with the original number, then it results in all digits from 1 to 9 present exactly once.
+
+## Examples
+
+Example 1:
+
+Input: N = 192
```
Comment (Ajay-Dhangar) [inline dsa-solutions/gfg-solutions/0002-fascinating-number.md:27]: Bad practice:

```md
Input: N = 192
Output: Fascinating
Explanation: After multiplication with 2 and 3, and concatenating with original number, number will become 192384576 which contains all digits from 1 to 9.
```

Good Practice:

````md
```
Input: N = 192
Output: Fascinating
Explanation: After multiplication with 2 and 3, and concatenating with original number, number will become 192384576 which contains all digits from 1 to 9.
```
````

File: dsa-solutions/gfg-solutions/0002-fascinating-number.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+## Your Task
+
+You don't need to read input or print anything. Your task is to complete the function `fascinating()` which takes the integer n parameters and returns boolean (True or False) denoting the answer.
+
+Expected Time Complexity: $O(1)$
+Expected Auxiliary Space: $O(1)$
+
+## Constraints
+
+100 <= N <= 2*10^9
```
Comment (Ajay-Dhangar) [inline dsa-solutions/gfg-solutions/0002-fascinating-number.md:46]: bad practice:

```md
100 <= N <= 2*10^9
```

Good Practice:

```md
`100 <= N <= 2*10^9`
```

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
Change code for good practice

**Comment by iamanolive** [comment]
@Ajay-Dhangar made the changes

**Comment by iamanolive** [comment]
just asking for future reference, but what could i have done better in this for it to qualify as a level 3? @Ajay-Dhangar

**Review by Ajay-Dhangar — APPROVED** [review]
Done

**Comment by Ajay-Dhangar** [comment]
> just asking for future reference, but what could i have done better in this for it to qualify as a level 3? @Ajay-Dhangar

Please follow our structure, similar to the source code of the [Two Sum problem](https://github.com/CodeHarborHub/codeharborhub/blob/main/dsa-solutions/lc-solutions/0000-0099/0001-two-sum.md?plain=1), to ensure better understanding for everyone. You can also refer to the [live preview](https://www.codeharborhub.live/dsa-solutions/lc-solutions/0000-0099/two-sum) for more details.

If you create the content according to this structure, I will assign it a level3 rating. However, if you cannot meet this requirement, I will need to update the content based on my specifications and assign you level2.

**Comment by Ajay-Dhangar** [comment]
> > just asking for future reference, but what could i have done better in this for it to qualify as a level 3? @Ajay-Dhangar
> 
> Please follow our structure, similar to the source code of the [Two Sum problem](https://github.com/CodeHarborHub/codeharborhub/blob/main/dsa-solutions/lc-solutions/0000-0099/0001-two-sum.md?plain=1), to ensure better understanding for everyone. You can also refer to the [live preview](https://www.codeharborhub.live/dsa-solutions/lc-solutions/0000-0099/two-sum) for more details.
> 
> If you create the content according to this structure, I will assign it a level3 rating. However, if you cannot meet this requirement, I will need to update the content based on my specifications and assign you level2.

I am waiting for your response. Would you like to update your content, or should I go ahead and merge it?

**Comment by iamanolive** [comment]
> > > just asking for future reference, but what could i have done better in this for it to qualify as a level 3? @Ajay-Dhangar
> > 
> > 
> > Please follow our structure, similar to the source code of the [Two Sum problem](https://github.com/CodeHarborHub/codeharborhub/blob/main/dsa-solutions/lc-solutions/0000-0099/0001-two-sum.md?plain=1), to ensure better understanding for everyone. You can also refer to the [live preview](https://www.codeharborhub.live/dsa-solutions/lc-solutions/0000-0099/two-sum) for more details.
> > If you create the content according to this structure, I will assign it a level3 rating. However, if you cannot meet this requirement, I will need to update the content based on my specifications and assign you level2.
> 
> I am waiting for your response. Would you like to update your content, or should I go ahead and merge it?

I would like to update it

**Comment by iamanolive** [comment]
@Ajay-Dhangar made changes

**Comment by Ajay-Dhangar** [comment]
> @Ajay-Dhangar made changes

I check it.

**Review by Ajay-Dhangar — APPROVED** [review]
Okay,

**Pull request merged**